index.html

```diff
@@ -316,39 +316,52 @@ <h3>
             <tr>
               <th>Registry</th>
               <th>Description</th>
+              <th>Based on</th>
             </tr>
             <tr>
               <td>
-<a href="https://w3c-ccg.github.io/vc-extension-registry/">Verifiable Credential Extensions Registry</a>
+Verifiable Credential Extensions Registry
```

Comment:
Per my earlier review, "Verifiable Credential Extensions Registry" is almost certainly wrong, since each kind of VC extension will need its own registry.  Please delete this overly-specific and misleading entry.  Compare this to https://www.iana.org/assignments/oauth-parameters/oauth-parameters.xhtml, which defines eleven extensions registries for OAuth.

Reply:
Fixed in fb17a5abed3d92b62ed4d4d918d81a80426e56de and c47ffd50ad6224d41e6bc7deabb627aef7a78db2.